src/sentry/integrations/slack/tasks.py

```diff
@@ -170,6 +180,33 @@ def find_channel_id_for_alert_rule(organization_id, uuid, data, alert_rule_id=No
             redis_rule_status.set_value("failed")
             return
 
+    # This function call will write "input_channel_id" to the appropriate actions before we create the serializer
+    # I wasn't sure of a way to use the same loop to return True in the endpoints and to also do the lookup and didn't want to duplicate the loop
+    try:
+        mapped_ids = get_slack_channel_ids(organization, user, data)
+    except (serializers.ValidationError, ChannelLookupTimeoutError, InvalidTriggerActionError):
+        # channel doesn't exist error or validation error
+        logger.info(
+            "get_slack_channel_ids.failed",
+            extra={
+                "user_id": user.id if user is not None else None,
+                "organization_id": organization_id,
+                "data": data,
```

Comment:
@iProgramStuff I don't know if passing in a dictionary for `data` here will work as expected (at worst, the log message won't show up in Kibana). I don't think we need that in the log message. Just knowing which org had which error is enough for me.

Reply:
I just pass the exception now which we seem to do with lots of other logs

Reply:
@iProgramStuff did you push your changes?